moto/ec2/models/elastic_network_interfaces.py

```diff
@@ -75,6 +75,8 @@ def __init__(
         self.public_ip_auto_assign = public_ip_auto_assign
         self.start()
         self.add_tags(tags or {})
+        self.requester_managed = False
+        self.requester_id = "AIDAIT2UOQQY3AUEMOTO"
```

Comment:
I don't see `RequesterId` showing up in real AWS responses unless `RequesterManaged` is `True`, do you?

Reply:
Yep, I see both with `RequesterManaged=False`.